packages/patternfly-4/react-core/src/components/Title/Title.tsx

```diff
@@ -12,14 +12,14 @@ export interface TitleProps extends Omit<React.HTMLProps<HTMLHeadingElement>, 's
   /** Additional classes added to the Title */
   className?: string;
   /** the heading level to use */
-  headingLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+  level: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
```

Comment:
Should it be required if we anyways default it to h1?

Reply:
It should be required, that's really the primary change for this PR. I wasn't able to remove 'h1' as a default value though, I don't understand why. When I run the `build:docs` target with that in place, it blows up like this;

<img width="1661" alt="Screen Shot 2019-04-09 at 3 36 45 PM" src="https://user-images.githubusercontent.com/5942899/55829733-61682c00-5add-11e9-9c3f-fc78dbcdc6e6.png">

I spent a while trying to debug it but didn't wind up with anything useful. In any case, the prop is required so we have the expected experience on the consuming side and the default value seems to have no effect other than silencing the build failure. A code smell, I know.

WDYT? If you have an idea on how to fix this I'd love to hear. The only way I know to get around it is using the `createElement` function but that's been advised against so I'm open to suggestion.